packages/agoric-cli/package.json

```diff
@@ -23,9 +23,9 @@
   },
   "dependencies": {
     "@agoric/bundle-source": "^1.0.3",
-    "@agoric/captp": "^1.0.2",
+    "@agoric/captp": "^1.2.0",
     "@agoric/evaluate": "^2.1.3",
```

Comment:
The `evaluate` dependency remains at `2.1.3`, but `evaluate` was upgraded to `2.2.0` . Other packages that depend upon `evaluate` had their dependencies increased. Should this one be increased too? Or can `agoric-cli` use the older `evaluate` without problems?

Reply:
Should be increased.  I've done that now.